pkg/network/http/ebpf_main.go

```diff
@@ -65,29 +84,33 @@ func newEBPFProgram(c *config.Config) (*ebpfProgram, error) {
 		},
 	}
 
-	return &ebpfProgram{
+	for _, m := range mainHTTPMaps {
+		mgr.Maps = append(mgr.Maps, &manager.Map{Name: m})
```

Comment:
Why are these maps not defined declaratively? It doesn't seem like they are shared like the other maps.

Reply:
Some of them are shared and some of them are not. I thought it was better to declare them as global variable in terms of readability